src/sid/parse_model.py

```diff
@@ -56,7 +56,9 @@ def parse_share_known_cases(share_known_cases, duration, burn_in_periods):
             )
         # Extend series with burn-in periods and if shares for burn-in periods do not
         # exist, backfill NaNs.
-        share_known_cases = share_known_cases.reindex(extended_index).backfill()
+        share_known_cases = share_known_cases.reindex(extended_index).fillna(
+            method="bfill"
+        )
```

Comment:
Can you remember why the old version did not work? They should be the same.

Reply:
Yes. The other version requires a very new version of pandas that is not compatible with estimagic yet.